Dockerfile

```diff
@@ -45,7 +45,10 @@ ARG TARGETPLATFORM
 # lld has issues building static binaries for ppc so prefer ld for it
 RUN set -e; xx-apk add musl-dev gcc libseccomp-dev libseccomp-static; \
   [ "$(xx-info arch)" != "ppc64le" ] || XX_CC_PREFER_LINKER=ld xx-clang --setup-target-triple
-RUN --mount=from=runc-src,src=/usr/src/runc,target=. --mount=target=/root/.cache,type=cache \
+RUN --mount=from=runc-src,src=/usr/src/runc,target=.,rw --mount=target=/root/.cache,type=cache \
+  # FIXME: temporarily patch runc to fix libseccomp issue https://github.com/seccomp/libseccomp-golang/pull/85
```

Comment:
Should it use the other PR? Not sure. Just commented on one of the linked PRs;

> I'd love to see it merged in libseccomp; carried tonis's PR last week to try to unblock it; please review that one (hopefully it will help getting attention of the maintainers there) [seccomp/libseccomp-golang#90](https://github.com/seccomp/libseccomp-golang/pull/90)

The danger of taking a PR branch as reference, is of course that it will go away, so at some point no longer builds

Reply:
Yes I will switch to yours instead.

Reply:
Done. Can you protect this branch on your fork? :sweat_smile:

Reply:
Yeah we can keep in draft in the meantime. Hopefully maintainers of libseccomp-golang will merge that soon.

Reply:
https://github.com/opencontainers/runc/pull/3465 merged on runc. Use this commit and remove hacky go mod edit.